www/assets/locales/locale-en.json

```diff
@@ -294,10 +294,14 @@
       "title": "Statistics",
       "y-zero": "Begin Y Axis at Zero",
       "average-line": "Show Average Line",
-      "goal-line": "Show Goal Line",
-      "send-data": "Send data to external service",
-      "send-data-address": "Service Address",
-      "send-data-authorization": "Service Authorization Header"
+      "goal-line": "Show Goal Line"
+    },
+    "developer": {
+      "data-sharing": "Send data to external service",
```

Comment:
A `title` string is missing here.

Reply:
I will fix this tomorrow. I will also test this version on Android Studio and remove the "WIP" flag.